plutus-core/plutus-core/src/PlutusCore/Builtins.hs

```diff
@@ -267,6 +273,7 @@ instance Serialise DefaultFun where
               Nop1                 -> 25
               Nop2                 -> 26
               Nop3                 -> 27
+              EqualsString         -> 28
```

Comment:
I'd suggest moving this up to before `Nop1`.  The `Nop` builtins are only there temporarily and we plan to remove them soon (https://jira.iohk.io/browse/SCP-2267), so it'd be good if removing them doesn't make anything else move.

Reply:
... and the same for the other occurrences in this file.

Reply:
Rearranged 👌